tests/ui/impl-trait/indirect-impl-for-trait-obj-coherence.rs

```diff
@@ -0,0 +1,28 @@
+// check-pass
```

Comment:
This test isn't in the right place (`impl-trait` tests are for `impl Trait` syntax).

Maybe just under `tests/ui/coherence`

Reply:
Thanks, moved to `tests/ui/coherence` as suggested.